cxplat/src/cxplat_winkernel/memory_winkernel.c

```diff
@@ -56,3 +57,40 @@ cxplat_free(_Frees_ptr_opt_ void* pointer, cxplat_pool_flags_t pool_flags, uint3
         ExFreePoolWithTag(pointer, tag);
     }
 }
+
+_Must_inspect_result_ cxplat_status_t
+cxplat_allocate_lookaside_list(
+    _Out_ cxplat_lookaside_list_t* lookaside, _In_ size_t size, _In_ uint32_t tag, _In_ cxplat_pool_flags_t pool_flags)
```

Comment:
```suggestion
    _Out_ cxplat_lookaside_list_t* lookaside, size_t size, uint32_t tag, cxplat_pool_flags_t pool_flags)
```
`_In_` is for reference types not value types